contrib/completion/bash/docker

```diff
@@ -3493,6 +3494,10 @@ _docker_service_update_and_create() {
 			__docker_nospace
 			return
 			;;
+		--isolation)
+			COMPREPLY=( $( compgen -W "default process hyperv" -- "$cur" ) )
+			return
+			;;
```

Comment:
There is existing code to handle this completion, see https://github.com/docker/cli/blob/master/contrib/completion/bash/docker#L2496-L2500.